ts/server.go

```diff
@@ -89,8 +89,9 @@ func (s *Server) handleQuery(w http.ResponseWriter, r *http.Request, _ httproute
 		response.Results = append(response.Results, &TimeSeriesQueryResponse_Result{
 			Name:             q.Name,
 			Sources:          sources,
-			SourceAggregator: q.SourceAggregator,
-			Downsampler:      q.Downsampler,
+			SourceAggregator: q.GetSourceAggregator().Enum(),
```

Comment:
Seems like we want these enums to be non-nullable. This is odd, I thought we had disabled generating these getters...